scraper.rb

```diff
@@ -2,12 +2,16 @@
 # encoding: utf-8
 
 require 'wikidata/fetcher'
+require 'pry'
 
 en = WikiData::Category.new('Category:Members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:Women members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:People\'s Democratic Party members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:Members of the House of Representatives (Nigeria) from Rivers State', 'en').member_titles |
-     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles 
+     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles
 
-EveryPolitician::Wikidata.scrape_wikidata(names: { en: en })
+sparq = 'SELECT ?item WHERE { ?item p:P39 ?position_held . ?position_held ps:P39 wd:Q21290864 . ?position_held pq:P2937 wd:Q20311259 }'
```

Comment:
This is a little too complex to have on a single line, I fear. For legibility, and consistency with all the other scrapers where we do equivalent queries (and even just for it to be more usable on https://query.wikidata.org/, rather than most of it to be covered by the query explainer), it would be better to split this over multiple lines.

![screen shot 2017-03-07 at 09 36 04](https://cloud.githubusercontent.com/assets/57483/23650290/8c041e10-0319-11e7-9808-169449fe8522.png)